src/lib/viewers/controls/icons/IconDrawing24.tsx

```diff
@@ -1,6 +1,10 @@
 import * as React from 'react';
 
-function IconDrawing24({ fill, ...rest }: React.SVGProps<SVGSVGElement>): JSX.Element {
+type Props = React.SVGProps<SVGSVGElement> & {
+    fill?: string;
```

Comment:
I think `fill` is a valid SVG prop, so we shouldn't need to redeclare it. Did the underlying type change?

Reply:
Ah, hm, looks like `SVGProps` is an interface, perhaps it's similar to the other props issue that came up

Reply:
Per the slack discussion, we can disable this rule for TS files for now - seems like it's a known issue and after looking at interfaces, it seems like we do have the fill property; so it looks like this is a false positive.